src/Compilers/CSharp/Portable/BoundTree/Statement.cs

```diff
@@ -23,7 +23,7 @@ internal partial class BoundBadStatement
 
     partial class BoundLocalFunctionStatement
     {
-        protected override ImmutableArray<BoundNode> Children => ImmutableArray.Create<BoundNode>(this.Body);
+        protected override ImmutableArray<BoundNode> Children => ImmutableArray.Create<BoundNode>(this.BlockBody, this.ExpressionBody);
```

Comment:
>protected override ImmutableArray<BoundNode> Children => ImmutableArray.Create<BoundNode>(this.BlockBody, this.ExpressionBody); [](start = 8, length = 127)

I think this code should be removed, we have support for local functions. See the comment in the base class above.